proceeding/tables/image_table.tex

```diff
@@ -0,0 +1,22 @@
+%table
+
+\begin{table}[h]
+
+\caption{Image information.}
+\label{tab:a}
+\tabcolsep7pt\begin{tabular}{ || lrlll ||}
+\hline
+
+\textbf{Image} & \textbf{Resolution (arcsec)} & \textbf{Type} & \textbf{Color?} & \textbf{Description}\\ \hline
+\textbf{Fermi color} & 51.53 & gamma-ray & color & Fermi-LAT\\
+\textbf{AKARI 90um} & 51.53 & infrared & grayscale & AKARI\\
+\textbf{CGPS-VGPS CONT} & 51.53 & radio & grayscale & \\
+\textbf{Spitzer GLIMPSE360} & 1.2 & infrared & color & Spitzer \\
+\textbf{Haslam 408} & 51.53 & radio & grayscale & 408 MHz\\
+\textbf{IRIS Band 4-100um} & 51.53 & infrared & grayscale & IRIS \\
+\textbf{Planck R2 LFI Color 30-44-70 GHz} & 51.53 & microwave & color & Planck 30-44-70 GHz\\
+\textbf{Planck R1 + R2 HFI Color 353-545-857 GHz} & 51.53 & microwave & color & Planck 353-545-857 GHz\\
```

Comment:
I agree, these are way too long.
Invent your own short label (like you did for the catalogs), and put all other info (if really needed) in the "description".